configuring/configuring-healthwatch.html.md.erb

```diff
@@ -529,6 +531,24 @@ To configure the **Resource Config** pane:
 
 1. Click **Save**.
 
+## <a id='resource-config'></a> (Optional) Configure for OpenTelemetry
+
+Follow these steps to configure HealthWatch and receive data through the OpenTelemetry collector for Tanzu Application Service 6.0 and above:
+
+1. Click the Settings tab and select **Resource Config**.
+    1. Set the **TAS Configure Exporter** and **TAS Gauge Exporter** configurations to `O` because they are only used by FireHose.
+    1. If you made any changes, click **Save**.
+1. Click the **Certificates** tab:
+    1. Click **Link to Credential** next to **Healthwatch Exporter Client Mtls**.
```

Comment:
This will change based on the new config that we add for certificates

Reply:
Thank you! Please let me know once it is decided.